gulp-tasks/lint.js

```diff
@@ -0,0 +1,14 @@
+const gulp = require('gulp');
+const exec = require('child-process-promise').exec;
+
+const logHelper = require('./utils/log-helper');
+
+// Use npm run lint to ensure we are using local eslint
+gulp.task('lint', () => {
```

Comment:
You're right, it's much better. Just moved to `gulp-tasks/utils/spawn-promise-wrapper.js`